common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -176,14 +183,23 @@ def metadata_for_mode(self, player_mode, additional_data=None):
             })
 
         if player_mode == 'html5_unsupported_video':
+            metadata.update(youtube_ids)
             metadata.update({
-                'youtube_id_1_0': '',
-                'youtube_id_0_75': '',
-                'youtube_id_1_25': '',
-                'youtube_id_1_5': '',
                 'html5_sources': HTML5_SOURCES_INCORRECT
             })
 
+        if player_mode == 'hls':
+            metadata.update(youtube_ids)
```

Comment:
In the absence of these youtube ids, default youtube id(`3_yD_cEKoCk`) will be picked and due to this `youtube` video will played.